docs/nodes/AI_ML/CLASSIFICATION/ACCURACY/ACCURACY.md

```diff
@@ -43,17 +43,15 @@ import outputImg from './examples/EX1/output.jpeg'
 <Example1 />
 
 <SectionBreak />
-  
+
 [//]: # (Appendix)
 
+## Appendix
```

Comment:
Have you checked on the rendered site that this puts the heading in the right location?